packages/frontend/src/utils/requestUtils.ts

```diff
@@ -78,7 +81,7 @@ export const getResultsFromStream = async <T>(url: string | undefined) => {
     }
 };
 
-// recursively check job status until it's complete
+// TODO: this + type should be named something more generic since it is reused in multiple features
```

Comment:
moved the file here and added a comment about the names @IrakliJani I don't think this should be done in this PR